packages/server/lib/middleware/security.ts

```diff
@@ -24,7 +24,16 @@ export function securityMiddlewares(): RequestHandler[] {
             directives: {
                 defaultSrc: ["'self'", hostPublic, hostApi],
                 childSrc: "'self'",
-                connectSrc: ["'self'", 'https://*.google-analytics.com', 'https://*.sentry.io', hostPublic, hostApi, hostWs.href, 'https://*.posthog.com'],
+                connectSrc: [
+                    "'self'",
+                    'https://*.google-analytics.com',
+                    'https://*.sentry.io',
+                    hostPublic,
+                    hostApi,
+                    hostWs.href,
+                    'https://*.posthog.com',
+                    'https://bluegrass.nango.dev'
```

Comment:
*.nango.dev or use the PUBLIC_KOALA_CDN_URL hostname (cf hostWs) 
don't forget the code is shared with opensource so stuff like this should not be hardcoded (even if there is no security problem)

Reply:
:+1: I was thinking that but then was also surprised about it opening up to Google and posthog.

Reply:
See 96c4eb4c3